src/Cassandra/Mapping/PocoDataFactory.cs

```diff
@@ -67,8 +66,7 @@ public void AddDefinitionDefault(Type type, Func<ITypeDefinition> definitionHand
         private PocoData CreatePocoData(Type pocoType)
         {
             // Try to get mapping from predefined collection, otherwise fallback to using attributes
-            ITypeDefinition typeDefinition;
-            if (_predefinedTypeDefinitions.TryGetItem(pocoType, out typeDefinition) == false)
+            if (_predefinedTypeDefinitions.TryGetItem(pocoType, out ITypeDefinition typeDefinition) == false)
```

Comment:
```suggestion
            if (!_predefinedTypeDefinitions.TryGetItem(pocoType, out ITypeDefinition typeDefinition))
```

Reply:
Generally for methods I find it much more robust to use ` == (boolean)`

For sample:
```csharp
{
    bool? getTryValue() => false;

    if (!getTryValue()) { } //Error
    if (getTryValue() == false) { } //Success
}
```

Reply:
I understand the argument for that approach for sure but we should use `!` here to keep it consistent.